Player/Player.gd

```diff
@@ -62,6 +62,8 @@ func _ready():
 	#giveBodypart(GlobalRegistry.createBodypart("caninepenis"))
 	updateNonBattleEffects()
 	
+	skillsHolder.addSkillExperience(Skill.Start, 0)
```

Comment:
This is very hack-ish. What if `addSkillExperience` changes so it doesn't add the skill if the exp is 0. I'd prefer some kind of `alwaysVisible()` func or something for the skill then. But I'm still not sure if showing a skill that doesn't have anything unlockable is a good idea. Player might assume that all skills will be like that